.github/workflows/Line_check.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```

Comment:
I think this file should be removed, as we have an official suite of tests that will be run.

Reply:
**Action item:** Please remove this file.